README.md

```diff
@@ -1,2 +1,78 @@
 # transactionfee-info
-Bitcoin Protocol Layer Statistics
+This project provides protocol-level statistics and insights about Bitcoin: Blocks, Transactions, SegWit and more. Implemented with a Rust backend and Hugo-based frontend.
+
+## Prerequisites
+- Nix
+- Access to a Bitcoin Core node with REST API enabled
+
+### Bitcoin Core Setup
+1. Add the following lines to your `bitcoin.conf`
+2. Restart Bitcoin Core
+
+```conf
+# Enable REST API
+rest=1
+```
+
+## Quick Start with Nix
+With Nix installed enter a nix-shell to setup and start the backend and frontend
+
+```bash
+# Enter the development environment
+nix-shell
+```
+
+### Backend
+
+```bash
+# Once in nix-shell..
+
+# Setup and start the backend
+cd backend
+# Run database migrations
+diesel setup
+diesel migration run
+# This uses the default settings of connecting to localhost:8332
+# creating a SQLite database in ./db.sqlite
+# and writing the CSV files to ./csv
+# See below for more details on the command-line arguments
+cargo run --release &
```

Comment:
I think it's clearer to users that this process is still running if we don't suggest adding a `&`
```suggestion
cargo run --release
```